frontend/public/components/monitoring/query-browser.tsx

```diff
@@ -576,7 +574,7 @@ const QueryBrowser_: React.FC<QueryBrowserProps> = ({
   // For the default time span, use the first of the suggested span options that is at least as long
   // as defaultTimespan
   const defaultSpanText = spans.find((s) => parsePrometheusDuration(s) >= defaultTimespan);
-
+  const { t } = useTranslation();
```

Comment:
Minor: Should probably leave this line at the top of the component

Reply:
Done,move it to the top of the component. Thanks.